src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.skia.cs

```diff
@@ -1538,9 +1535,9 @@ private void OnInvalidateRender()
 
 		public void Hide()
 		{
-			if (XamlRoot is { })
+			if (Visual.CompositionTarget is CompositionTarget target)
 			{
-				XamlRoot.RenderInvalidated -= OnInvalidateRender;
+				target.FramePainted -= OnFramePainted;
```

Comment:
Let's use `Render` (render to SKPicture) and `Draw` (SKPicture to screen) for the vocabulary.